src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -77,6 +82,96 @@ def get_category_for(unit: Optional[str]) -> Optional[str]:
         return UnitDatabase.GetSingleton().GetDefaultCategory(unit)
 
 
+def update_multi_input_flags(
+    alfacase_content: DescriptionDocument, item_description: T
```

Comment:
I used that for consistency.
Most of loaders use `document` or `alfacase_content`. While checking the code to better reply this comment I realized most time (maybe always) I call this function the current scope variable is `document`.
But `document` can be a bit misleading since it ussually is a fragment (not the full alfacase document).

I will update it to use the same name as the call sites (just `document`).